api/config/cdr_config_local.json

```diff
@@ -31,36 +31,33 @@
       "creationTime": "2018-06-06 00:00:00Z",
       "releaseNumber": 1,
       "numParticipants": 0,
-      "cdrDbName": "cdr",
+      "cdrDbName": "synth_r_2020q1_3",
```

Comment:
`cdrVersions` now exactly matches the Test environment.

`accessTiers` does not, and this is intentional: we want them to have different `authDomainGroupEmail` values.

Reply:
This will not work as local is only setup to support 1 version of the mysql database in docker. When running dev-up locally, I get the following error when trying to access the mysql instance:
![Screen Shot 2022-11-29 at 8 59 12 AM](https://user-images.githubusercontent.com/3904919/204564093-7fa99056-2756-4576-87b8-6188c57b3ea5.png)